backend/app/views.py

```diff
@@ -517,7 +517,8 @@ def photographer_list_view(request):
     """
     Photographer list page
     """
-    photos_dir = os.path.join(settings.LOCAL_PHOTOS_DIR, 'photographers')
+    # photos_dir = os.path.join(settings.LOCAL_PHOTOS_DIR, 'photographers')
```

Comment:
Don't leave dead code in PRs -- if you've made a change, please do go ahead and delete the code that's replaced.